src/main/java/net/mcreator/ui/action/ActionRegistry.java

```diff
@@ -253,6 +254,7 @@ public ActionRegistry(MCreator mcreator) {
 				UIRES.get("16px.importstructure"));
 		this.importStructureFromMinecraft = new StructureImportActions.ImportStructureFromMinecraft(this);
 		this.importJavaModel = new ModelImportActions.JAVA(this).setIcon(UIRES.get("16px.importjavamodel"));
+		this.importGeckoLibModel = new ModelImportActions.GECKOLIB(this);

```

Comment:
Icon? Matej?

Reply:
![image](https://user-images.githubusercontent.com/38427877/150647429-88e69dd2-7d1f-41f6-8ecb-c1fb300ad553.png)
I made this logo for the moment. I look the same way the other ones and we see what's model type it is.